### ISSUE TITLE
Registry URL for MySql 8.0 not working

### ISSUE THREAD

**Issue by imranrazakhan** [sclorg/mysql-container#243]
I was reading readme on `https://github.com/sclorg/mysql-container/tree/master/8.0`, but the registry URL mentioned here are not working

```
registry.access.redhat.com/rhscl/mysql-80-rhel7
https://hub.docker.com/r/centos/mysql-80-centos7/
```

**Comment by omron93** [comment]
`mysql-80-rhel7` is not released yet. It is mentioned in README because it's expected to be released and it's easier to have READMEs in all versions similar.

CentOS7 image in Docker Hub should be fixed by #244

**Comment by phracek** [comment]
It is already released. CLosing this issue.
